pkgs/development/libraries/mapnik/cmake-harfbuzz.patch

```diff
@@ -0,0 +1,24 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 47bbaa15d..f04bbae88 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -155,17 +155,8 @@ endif()
+ 
+ mapnik_find_package(Freetype REQUIRED)
+ 
+-# try to find harfbuzz with the native configuration and fallback to our "own" FindHarfBuzz
+-mapnik_find_package(harfbuzz CONFIG QUIET)
+-if(harfbuzz_FOUND)
+-    message(STATUS "Found harfbuzz native cmake")
+-    list(APPEND MAPNIK_OPTIONAL_LIBS harfbuzz::harfbuzz)
+-else()
+-    # we use our "own" FindHarfBuzz. See https://github.com/mapnik/mapnik/pull/4191#issuecomment-874728157 for more details
+-    message(STATUS "Fallback to FindHarfBuzz")
+-    mapnik_find_package(HarfBuzz ${HARFBUZZ_MIN_VERSION} REQUIRED COMPONENTS ICU)
+-    list(APPEND MAPNIK_OPTIONAL_LIBS HarfBuzz::HarfBuzz HarfBuzz::ICU)
+-endif()
++pkg_check_modules(harfbuzz REQUIRED IMPORTED_TARGET harfbuzz)
++list(APPEND MAPNIK_OPTIONAL_LIBS PkgConfig::harfbuzz)
```

Comment:
With https://github.com/mapnik/mapnik/pull/4270 merged, you can pass `CMAKE_DISABLE_FIND_PACKAGE_harfbuzz=OFF` to the configure call to disable the `find_packge(harfbuzz)` and force pkg-config therefore. So the patch can be removed after that

Reply:
That's nice!
Any chance we'd get a new mapnik release anytime soon? Using master in Nixpkgs is usually only done for special cases, e.g. when the last stable release is broken/unsecure.

Reply:
Good question. I don't really know which plans @artemp has regarding mapnik 4.0.0. I've just started contributing first things to mapnik like the CMake build and windows support.

Reply:
merged. Patch could be removed